manifests/server/service.pp

```diff
@@ -33,6 +34,7 @@
         run_as          => $user,
         database_name   => $default_database,
         database_port   => $port,
+        connect_settings => $connect_settings,
```

Comment:
@vinzent , looks like the linter is complaining about the hash rocket alignment in this resource. please make sure they all line up.